### PR TITLE
Update NEWS.md with changes since 1.7

### PR DESCRIPTION
**Pull request by wader**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+## Language changes
+
+- Simplify `paths/0` and `paths/1`. @asheiduk @emanuele6 #2946
+- Reject `U+001F` in string literals. @torsten-schenk @itchyny @wader #2911
+- Remove unused nref accumulator in `block_bind_library`. #2914
+- Remove a bunch of unused variables, and useless assignments. @emanuele6 #2914
+- main.c: Remove unused EXIT_STATUS_EXACT option. @emanuele6 #2915
+- Actually use number correctly casted from double to int as index. @emanuele6 #2916
+- src/builtin.c: remove unnecessary jv_copy-s in type_error/type_error2. @emanuele6 #2937
+- Remove undefined behavior caught by LLVM 10 UBSAN. @Gaelan @emanuele6 #2926
```
Comment (wader) [inline NEWS.md:19]: Too detailed list?

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
+# TBD
+
+TODO: Some text?
+
+# CLI changes
+
+- Make default background color more suitable for bright backgrounds. @mjarosie @taoky @nicowilliams @itchyny #2904
+- Comment bug fixes, and fully support Tcl-style multiline comments. @emanuele6 #2942
```
Comment (emanuele6) [inline NEWS.md]: this is a language change

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+- Allow passing the inline jq script after `--``. @emanuele6 #2919
```
Comment (emanuele6) [inline NEWS.md:9]: this is supposed to say "before"; the commit message is wrong :)

File: NEWS.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+- Comment bug fixes, and fully support Tcl-style multiline comments. @emanuele6 #2942
+- Allow passing the inline jq script after `--``. @emanuele6 #2919 
+- Restrict systems operations on OpenBSD and remove unused `mkstemp`. @klemensn #2934
+- Fix possible uninitialised value dereference if `jq_init()` fails. @emanuele6 @nicowilliams #2935
+
+## Language changes
+
+- Simplify `paths/0` and `paths/1`. @asheiduk @emanuele6 #2946
+- Reject `U+001F` in string literals. @torsten-schenk @itchyny @wader #2911
+- Remove unused nref accumulator in `block_bind_library`. #2914
```
Comment (emanuele6) [inline NEWS.md]: this is the only line without mentions

File: NEWS.md (hunk 2 above)
Comment (wader) [reply NEWS.md]: fixed

File: NEWS.md (hunk 4 above)
Comment (wader) [reply NEWS.md]: fixed

**Comment by wader** [comment]
Removed `Comment bug fixes, and fully support Tcl-style multiline comments. @emanuele6 #2942` and i guess we should eventually add note about GHSA-686w-5m7m-54vc

**Pull request merged**